scripts/gh.js

```diff
@@ -1,93 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-//  High-level github automation API.
```

Comment:
I renamed this to "ghlib" to clear up some ugliness during import/instantiation, where he'd have things like "gh = new GH.GH()".